src/index.d.ts

```diff
@@ -685,6 +685,11 @@ declare namespace RamdaAdjunct {
             (value: any): (obj: object) => boolean;
         };
 
+        /** 
+         * Checks if input value falls within the supplied inclusive range.
+         */
+        inRange(low: number, high: number, value: number): boolean;
```

Comment:
I'd add curried typings, at least this one:

```ts
inRange(low: number, high: number): (value: number) => boolean;
```

I'd also add typings in tests for `Date`s, they're coerced into ms timestamp and comparison operators work corrently with them.

```ts
inRange(low: Date, high: Date, value: Date): boolean;
inRange(low: Date, high: Date): (value: Date) => boolean;
```